src/service/holdings.ts

```diff
@@ -18,8 +20,8 @@ import { TokenModel, Tokens } from "./Data"
 import { ProviderResult } from "src/utils/ProviderResult"
 import { Token } from "@celo/contractkit"
 import addressesConfig from "src/addresses.config"
-import { getCurveUSDC } from "src/providers/Celo"
 import { allOkOrThrow, ResultOk, valueOrThrow } from "src/utils/Result"
+//import { BTC_AXELAR_ADDRESS, BTC_WORMHOLE_ADDRESS, CELO_ADDRESS, ETH_AXELAR_ADDRESS, ETH_WORMHOLE_ADDRESS, RESERVE_MULTISIG_CELO, STAKED_CELO_ERC20_ADDRESS } from "src/contract-addresses"
```

Comment:
Remove?

Reply:
those are going to be needed once the calls to uniV3HoldingsForToken() are uncommented if we don't wont to hardcode the token addresses in those calls

Reply:
Ah ok, thought it was something else that was not needed